modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js

```diff
@@ -20,6 +20,7 @@
 var resources = function () {
     return {
         css:['common.css','sidepanel.css', 'left-navigation.css', 'font-wso2.css', 'bootstrap-dialog.min.css', 'typeahead.css'],
-        js:['sidepanel.js','greg-subscriptions-api.js','bootstrap-dialog.min.js', 'typeahead.bundle.min.js', 'typeahead.js']
+        js:['sidepanel.js','greg-subscriptions-api.js','bootstrap-dialog.min.js', 'typeahead.bundle.min.js', 'typeahead.js'],
+        code: ['taxonomy-tenant-meta.hbs']
```

Comment:
Thank you Sameera for mentioning it,
This has been fixed with wso2/carbon-store#834